pennylane/io.py

```diff
@@ -410,6 +410,91 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`NoiseModel <pennylane.noise.NoiseModel>`.
+
+    .. note::
+
+        This function depends upon the PennyLane-Qiskit plugin. Follow the
+        `installation instructions <https://docs.pennylane.ai/projects/qiskit/en/latest/installation.html>`__
+        to get up and running. You may need to restart your kernel if you are running in a notebook
+        environment.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a ``NoiseModel`` constructed in Qiskit
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        quantum_error (bool): include quantum errors in the converted noise model. Default is ``True``.
+        readout_error (bool): include readout errors in the converted noise model. Default is ``True``.
```

Comment:
Maybe we don't include this argument until it is supported in PL?

Reply:
These were relics from the older codes and are no longer used.